### DISCUSSION TITLE
request for enhancement to index functions in discord channels

### DISCUSSION THREAD

**Discussion by uberdude420** [Kav-K/GPTDiscord#173] [General]
Any chance I could request an update to the discord channel index function in the sense of a setting to add a list of channels for it to watch to real-time update the discord index as chats happen potentially with a conversational channel to talk about the conversations had or posts scraped into various channels without having to use the /index querry
[9:51 AM]
Would be handy to index tutorial channels and the like and have it able to converse about how to use server functions and talk about chats had
[9:52 AM]
Or does it already watch channel after it's indexed? To continue updating the index

**Comment by Kav-K** [comment]
Unfortunately this would be a pretty tough architecture to implement, I dont think this is on the roadmap for the foreseeable future
